tests/model_serving/model_server/private_endpoint/test_kserve_private_endpoint.py

```diff
@@ -0,0 +1,75 @@
+from typing import Self
+
+from simple_logger.logger import get_logger
+from ocp_resources.namespace import Namespace
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.pod import Pod
+from ocp_resources.service_mesh_member import ServiceMeshMember
+from tests.model_serving.model_server.private_endpoint.utils import curl_from_pod
+
+
+LOGGER = get_logger(name=__name__)
+
+
+class TestKserveInternalEndpoint:
+    def test_deploy_model(
```

Comment:
i think it is better to split into 2 tests:
`test_deploy_model_state_loaded` and `test_deploy_model_url`